tests/src/providers/testqgsgdalprovider.cpp

```diff
@@ -257,5 +259,33 @@ void TestQgsGdalProvider::bandName()
   delete provider;
 }
 
+void TestQgsGdalProvider::bandNameNoDescription()
+{
+  QString raster = QStringLiteral( TEST_DATA_DIR ) + "/raster/band1_byte_ct_epsg4326.tif";
+  QgsDataProvider *provider = QgsProviderRegistry::instance()->createProvider( QStringLiteral( "gdal" ), raster, QgsDataProvider::ProviderOptions() );
+  QgsRasterDataProvider *rp = dynamic_cast< QgsRasterDataProvider * >( provider );
+  QVERIFY( rp );
+  if ( rp )
+  {
+    qDebug() << "Band Name 1: " << rp->generateBandName( 1 );
```

Comment:
Remove this - it's covered by QCOMPARE below, which will dump the obtained output if it differs